book/src/pod-inline-volume.md

```diff
@@ -0,0 +1,50 @@
+# Pod Inline Volume Support
+
+## Status
+
+Status | Min K8s Version | Max K8s Version 
+--|--|--
+Alpha | 1.15 | - 
+
+## Overview
+Traditionally, volumes that are backed by CSI drivers can only be used with a `PersistentVolume` and `PersistentVolumeClaim` object combination. This feature implements support that allows the nesting of volume declarations directly within pod specs for support of ephemeral storage using CSI drivers.  At runtime, nested inline volumes follow the ephemeral lifecycle of their associated pods where the driver handles all phases of volume operations as pods are created and destroyed.
```

Comment:
2nd sentence can maybe can be simplified to "This feature supports ephemeral storage use cases and allows CSI volumes to be specified directly in the pod specification."